services/graphite/README.md

```diff
@@ -40,16 +40,16 @@ matched multiple times. Multiple values will be joined together using the _Separ
 * Template: `.host.host.measurement.cpu.measurement`
 * Output: _measurement_ = `cpu.user` _tags_ = `host=localhost.localdomain cpu=cpu0`
 
-Since '.' requires queries on measurements to be double-quoted, you may want to set this to `_` to simplify querying parsed metrics.
+Since `.` requires queries on measurements to be double-quoted, you may want to set this to `_` to simplify querying parsed metrics.
 
 `servers.localhost.cpu.cpu0.user`
 * Separator: `_`
 * Template: `.host.measurement.cpu.measurement`
 * Output: _measurement_ = `cpu_user` _tags_ = `host=localhost cpu=cpu0`
 
-### Adding Tags
+### Adding tags
 
-Additional tags can be added to a metric that don't exist on the received metric.  You can add additional tags by specifying them after the pattern.  Tags have the same format as the line protocol.  Multiple tags are separated by commas.
+Additional tags can be added to a metric if they don't exist on the received metric.  You can add additional tags by specifying them after the pattern.  Tags have the same format as the plaintext protocol.  Multiple tags are separated by commas.
```

Comment:
line protocol was correct.  The format is the same as described in the [line protocol docs](https://docs.influxdata.com/influxdb/v1.0/write_protocols/line_protocol_reference/)